modules/1ad4goodBidAdapter.js

```diff
@@ -0,0 +1,493 @@
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER, NATIVE, VIDEO, ADPOD } from '../src/mediaTypes';
+import find from 'core-js/library/fn/array/find';
+import includes from 'core-js/library/fn/array/includes';
+
+const BIDDER_CODE = '1ad4good';
+const URL = '//hb.1ad4good.org/prebid';
+const VIDEO_TARGETING = ['id', 'mimes', 'minduration', 'maxduration',
+  'startdelay', 'skippable', 'playback_method', 'frameworks'];
+const USER_PARAMS = ['age', 'externalUid', 'segments', 'gender', 'dnt', 'language'];
+const APP_DEVICE_PARAMS = ['geo', 'device_id']; // appid is collected separately
+const DEBUG_PARAMS = ['enabled', 'dongle', 'member_id', 'debug_timeout'];
+const SOURCE = 'pbjs';
+const MAX_IMPS_PER_REQUEST = 15;
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['adsforgood', 'ads4good', '1adsforgood'],
+  supportedMediaTypes: [BANNER, VIDEO, NATIVE],
```

Comment:
Since, you don't support Native Ads yet, I suggest you remove Native.

Reply:
agreed, removed NATIV